index.html

```diff
@@ -0,0 +1,134 @@
+          </ul>
+          <div class="nav2">
+            <a class="logo-home" href="./index.html">
+            <img src="./images/icons/logo.png" alt="logo"/></a>
+            <ul>
+              <li><a href="./about.html">About</a></li>
+              <li><a href="#">News</a></li>
+              <li><a href="#">Videos</a></li>
+              <li><a href="#">Matches</a></li>
+              <li><a href="#">Teams</a></li>
+              <li><a href="#">Shop</a></li>
+            </ul>
+            <a class="zamalek-fc">Zamalek FC</a>
+          </div>
+        </div>
+        <div class="icon">
+          <div class="vector"></div>
+          <div class="vector"></div>
+          <div class="vector"></div>
+        </div>
+      </nav>
+    </header>
+    <main>
+      <section class="intro">
+        <p class="hello">"Hello! Zamalek fans"</p>
+        <h1>Election event for the most titled club in the 20th century.</h1>
+        <p class="header-description">
+          A general invitation to attend the Zamalek club elections for a new presidential term,
+          election festival , listen to candidates depates.
+        </p>
+        <p class="date">2022.11.10(THU) ~ 11(FRI)</p>
+        <p class="address">@ Cairo Nasser city zamalek club</p>
+      </section>
+      <section id="program" class="program">
+        <h2 class="program-title">Main program</h2>
+        <div class="program-cards">
+          <div class="card">
+            <img src="./images/teacher.png" alt="lecture-icon" />
+            <h3 class="card-title">Depates</h3>
+            <p>
+              Listen to candidates election programs about the next 4 years in the club.
+            </p>
+          </div>
+          <div class="card">
+            <img src="./images/board.png" alt="exhibition-icon" />
+            <h3 class="card-title">Election zone</h3>
+            <p>Discover your election zone</p>
+          </div>
+          <div class="card">
+            <img src="./images/text.png" alt="forum-icon" />
+            <h3 class="card-title">Discuss</h3>
+            <p>Have time to share your thoughts, and think about your choise</p>
+          </div>
+          <div class="card">
+            <img src="./images/pen.png" alt="workshops-icon" />
+            <h3 class="card-title">Elect</h3>
+            <p>
+              Go to your election zone , wait for your turn to make your choise.
+            </p>
+          </div>
+          <div class="card">
+            <img src="./images/net.png" alt="connect-icon" />
+            <h3 class="card-title">Results</h3>
+            <p>
+              gathering in the meeting room to announce the the results.
+            </p>
+          </div>
+        </div>
+        <div class="join-now">
+          <a class="save-spot" href="#">Save your spot for the Summit</a>
+          <a class="full-program" href="#">See the whole program</a>
+        </div>
+      </section>
+      <section class="candidates">
```

Comment:
- [ Optional ] The more button in the mobile version was supposed to display more speakers when clicked while the default number of speakers displayed on load is two. 
- [ Optional ] You may also add `less` button collapse back to two speakers when clicked.
![image](https://user-images.githubusercontent.com/44057863/169667630-072341bb-3130-401a-983b-2192679f9d28.png)

#### Design
![image](https://user-images.githubusercontent.com/44057863/169667652-a7e63d2e-a49e-4525-8d64-4e58a493c35e.png)